components/builder-graph/src/cyclic_package_graph.rs

```diff
@@ -77,4 +79,14 @@ impl PackageGraphTrait for CyclicPackageGraph {
     }
 
     fn stats(&self) -> Stats { self.graph.stats() }
+
+    fn compute_build(&self,
+                     touched: &[PackageIdentIntern],
+                     unbuildable: &dyn Unbuildable,
+                     _target: PackageTarget /* WRONG WRONG WRONG WRONG, WRONG WRONG WRONG
+                                             * WRONG */)
+                     -> PackageBuildManifest {
+        // TODO FIX assert!(_target == self.graph.current_target());
```

Comment:
Yeah, this would be nice to ensure always holds true.

Reply:
Fixed in 48ba35769a9de934379d165377e8cc9d0c708333